Attempted to add publish method

thekemkid opened the pull request:
I don't know if this will be accepted, but it's my run at the publish method which @geek mentioned. It's cool if this PR is closed, I just thought it might be handy to have this PR here as an option upon a positive decision. :smile: 

Resolves #219

geek commented on seneca.js:
```diff
@@ -796,6 +797,22 @@ function make_seneca (initial_options) {
+  // Perform an action, if the action exists. Don't kill seneca, unless the user
+  // provides a fatal$ flag.
+  function api_publish () {
+    var self = this
```
no need to set `self`, can use `this` in the function

thekemkid replied:
Will fix

geek commented on seneca.js:
```diff
@@ -796,6 +797,22 @@ function make_seneca (initial_options) {
+      args.fatal$ = false
+    }
+
+    return self.act(args, actdone)
```
do we care of the result, should we swallow it?

thekemkid replied:
2 thoughts:

It might be nice to give the user the chance to use a result if it exists.

and we don't gain much by swallowing it, do we? the user can ignore it, if they want to.

AdrieanKhisbe commented:
Not that convinced of the name `publish`.

thekemkid commented:
We have reverted this, and have broken the method out to a decorator, here:  https://github.com/senecajs/seneca-fire-and-forget

We renamed to `fire`.

AdrieanKhisbe commented:
Now you're talking! :D
